app/src/main/java/io/appium/uiautomator2/server/ServerInstrumentation.java

```diff
@@ -40,12 +40,13 @@
 import static io.appium.uiautomator2.server.ServerConfig.getServerPort;
 import static io.appium.uiautomator2.utils.Device.getUiDevice;
 
+
 public class ServerInstrumentation {
     private static final int MIN_PORT = 1024;
     private static final int MAX_PORT = 65535;
     private static final String WAKE_LOCK_TAG = "UiAutomator2:ScreenKeeper";
     public static final long MAX_TEST_DURATION = 24 * 60 * 60 * 1000;
-
+    public static Context ServerContext = null;
```

Comment:
the context must be retrieved from the Instrumentation singleton

Reply:
always try to avoid making public properties mutable, except of pure data classes.

Reply:
You mean i should create a method to retrieve the Context Right?